### DISCUSSION TITLE
Directus 9: Problem with translation

### DISCUSSION THREAD

**Discussion by yii-joblo** [directus/directus#3257] [Archive]
Here a the the steps

Languages
![image](https://user-images.githubusercontent.com/1770823/100763484-7e115000-33f5-11eb-95ae-1d7e5327fea6.png)

In need the collection "menu" translate - so I do autogenerate:
![image](https://user-images.githubusercontent.com/1770823/100763891-0b54a480-33f6-11eb-8633-b2870300a909.png)

![image](https://user-images.githubusercontent.com/1770823/100765098-676bf880-33f7-11eb-918d-3ed972374057.png)

I want the label translated

![image](https://user-images.githubusercontent.com/1770823/100764921-2d9af200-33f7-11eb-9a5e-f52fb3656674.png)
 

### But then I expected to see a dropdown with the languages for my "label", but I see a normal textinput.

![image](https://user-images.githubusercontent.com/1770823/100765790-2cb69000-33f8-11eb-83aa-5332361b8022.png)

In directus 8 it looks like:
![image](https://user-images.githubusercontent.com/1770823/100766385-e1e94800-33f8-11eb-87a6-6267fa5c2fa2.png)

What's wrong?

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/3230
